### ISSUE TITLE
swarm/storage/mru: `swarm resource help` outputs empty usage info

### ISSUE THREAD

**Issue by nolash** [ethersphere/swarm#786]
Branch `client-side-mru`

```
$ build/swarm resource help
NAME:
swarm resource - (Advanced) Create and update Mutable Resources

USAGE:
```

**Comment by jpeletier** [comment]
This happens with other commands as well, e.g., `swarm manifest`.
`swarm resource --help` shows help.

```
NAME:
   swarm resource - Works with Mutable Resource Updates

USAGE:
   swarm resource command [command options] <create|update|info>

COMMANDS:
     create  creates a new Mutable Resource
     update  updates the content of an existing Mutable Resource
     info    obtains information about an existing Mutable Resource

OPTIONS:
   --help, -h  show help
```

**Comment by nolash** [comment]
yeah, but shouldn't IMO

**Comment by jpeletier** [comment]
fixed in #788
